entrypoint.sh

```diff
@@ -13,12 +16,15 @@ initial_version=${INITIAL_VERSION:-0.0.0}
 tag_context=${TAG_CONTEXT:-repo}
 suffix=${PRERELEASE_SUFFIX:-beta}
 verbose=${VERBOSE:-true}
+branch_latest_commit=$BRANCH_LATEST_COMMIT
+use_last_commit_only=${USE_LAST_COMMIT_ONLY:-true}
 
-cd ${GITHUB_WORKSPACE}/${source}
+cd "${GITHUB_WORKSPACE}"/"${source}"
```

Comment:
"a"/"b" ? Why not "a/b" shellcheck

Reply:
@sbe-arg What breaking changes you refer to? As for not having `with_v` var set the behaviour is the same as it had been before - the param is optional with the default value of 'false'

Reply:
Fixed